module/VuFind/src/VuFind/Service/CurrencyFormatter.php

```diff
@@ -0,0 +1,106 @@
+<?php
+/**
+ * Currency formatter
+ *
+ * PHP version 7
+ *
+ * Copyright (C) Villanova University 2010.
```

Comment:
Should probably make this 2021.

Reply:
Well, it's mostly copy-paste from SafeMoneyFormat, so I thought it would be most appropriate to keep the old year.

Reply:
I usually update dates when I create new files, but I don't have strong feelings about this. No objections to leaving as-is. :-)